packages/renderer/src/lib/welcome/WelcomePage.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="ts">
+import bgImage from './background.png';
+import DesktopIcon from '../images/DesktopIcon.svelte';
+import Fa from 'svelte-fa/src/fa.svelte';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+import { onMount } from 'svelte';
+import { WelcomeUtils } from './welcome-utils';
+
+let showWelcome;
+
+$: showWelcome = false;
+
+onMount(async () => {
+  const welcomeUtils = new WelcomeUtils();
+  const ver = await welcomeUtils.getVersion();
+  if (ver === undefined) {
+    welcomeUtils.updateVersion('initial');
```

Comment:
From what I understand from your description of the PR
I guess we should store the version of Podman Desktop so it'll easy to know later what to while if we store 'initial' we may never know what to do later ?

Reply:
I thought of using the Podman Desktop version too, but if we didn't change this for a sprint or two I don't think we want to show it again every time users upgrade. I could use a different string like "v1" if you prefer.

Reply:
AFAIK if user upgrade the property is still stored so we won't show again the welcome page.
But then we could say that we need to display something on an upgrade while if we always store `initial` we don't really know 

It's a suggestion, not blocking the PR

Reply:
I would suggest the value stored should be the signature or last commit id of the welcome page